src/modules/IntModule.cpp

```diff
@@ -0,0 +1,58 @@
+#include "IntModule.h"
+#include <cstdio>
+
+IntModule::IntModule(ModularSynth& synth)
+:SynthModule(synth, moduleId, 0, 1, 1)
+{
+}
+
+
+void IntModule::cycle()
+{
+	setOutput(0, getParam(0));
```

Comment:
IMO if you wish to output int values you could do this like ``setOutput(0, static_cast<int>(getParam(0))`` to round the values to the nearest integer.